index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Portfolio</title>
+  <header>
+      <img src="img/My Logo.svg" alt="humburger" class="Logo">
+      <img src="img/Union.svg" alt="humburger"  class="Union" >
+  </header>
+</head><br><br><br><br>
+<body> 
+    <div class="container">
+      <h1>I'm Edgah.<br>
+        Glad to see<br>
+        you!</h1>
+      <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+    <img src="img/LET’S CONNECT.svg" alt="humburger"><br><br>
```

Comment:
- [ ] [OPTIONAL] As a software developer, your code must be scalable and maintainable. Imagine if you had to import images for all the text contents on this page instead of using semantic HTML tags to structure them. It will be very difficult, right? :thinking: 
  - Kindly consider using `h2.....h6` tag and probably style it as you would like to with correct `values` and `properties`